app/models/transaction.rb

```diff
@@ -49,6 +51,9 @@ def total
       where('bank_account_transactions.currency' => currency) }
   scope :incomes,     -> { joins(:category).where('categories.type' => Category::CATEGORY_INCOME)}
   scope :expenses,    -> { joins(:category).where('categories.type' => Category::CATEGORY_EXPENSE)}
+  scope :without_out, -> { joins('LEFT OUTER JOIN categories category_transactions
+      ON category_transactions.id = transactions.category_id').
+      where('category_transactions.id IS NULL OR category_transactions.name != ?', Category::CATEGORY_TRANSFER_OUTCOME) }
```

Comment:
@yunixon instead join you may only load id of transfer category and filter by id
```rb
where(category_id: Category.transfer_outcome_id)
```